docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,207 @@
+----
+galleon.sh install wildfly-servlet:current --dir=my-servlet-server
+----
+ 
+
+== Selecting the configurations to install
+
+Galleon allows to specify only a subset of the default configurations to be installed.
+
+To only install the standalone.xml configuration call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml
+----
+
+To install multiple configurations, a comma separated list of default configurations can be provided, 
+for example:
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml,domain/domain.xml
+----
+
+NB: A default configuration is identified by _<configuration model>_/_<configuration name>_. WildFly defines _standalone_ and _domain_ models. 
+The _configuration name_ is the XML configuration file name (eg: _standalone.xml_, _domain.xml_, _standalone-ha.xml_).
+
+== WildFly Galleon layers
+
+WildFly Galleon feature-packs expose some link:++https://docs.wildfly.org/galleon/#_layers++[Galleon layers]. 
+A layer identifies one or more server functionalities that can be installed on its own or 
+in combination with other layers. For example, if your application (eg: microservice) 
+is only making use of _jaxrs_ and _cdi_ server functionalities, you can choose to only install 
+the _jaxrs_ and _cdi_ layers. The _standalone.xml_ configuration would then only contain the 
+required subsystems and their dependencies.
+
+The benefit to install WildFly using Galleon layers, in addition to configuration 
+trim down, is that Galleon only install the needed content (JBOSS modules, scripts).
+
+To install a server only configured with jaxrs and cdi functionalities call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --layers=cdi,jaxrs
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to deploy an application
+that depends on _jaxrs_ and _cdi_. 
+
+NB:
+
+* Tools (jboss-cli, add-user, ...) are not installed when installing WildFly using layers. 
+Only server launcher scripts are installed into _bin_ directory.
+
+=== Defined Galleon layers
+
+==== Note on layers and feature-pack dependencies
+
+When installing a feature-pack, its own defined layers as well as the layers present in its dependencies 
+can be combined to build a custom configuration. This means that core and servlet defined layers can be combined with 
+WildFly defined layers to assemble a custom configuration. <<galleon-cli-recipes>> contains examples of such combination of layers.
+
+==== WildFly layers
+
+Basic layers:
+
+* _cdi_: Support for cdi.
+* _jaxrs_: Support for jaxrs.
+* _jms-activemq_: Support for connection to remote jms broker. 
+* _jpa_: Support for jpa (latest WildFly supported hibernate release).
+* _micro-profile_: Support for microprofile (config, health, metrics, open-tracing).
```

Comment:
Side issue (i.e. discussion beyond this PR):

Should this be called "observability"?

@jmesnil WDYT?

My concern is this can conflict with other ways we might handle MP features in the future; e.g. an FP from wildfly-extras or ....

Reply:
That depends how we want to decompose it:
* `microprofile` is fine if the layer is meant to provide *all* microprofile API we support. That'd be the equivalent of a `full-ee` layer
* `observability` is a better name to reflect the actual features provided by this layer (at least for `health, metrics, open-tracing`. `config` might belong to a dedicated layer if needs be).

I'd vote for `observability` if that's not too late to change it

Reply:
Renamed to be observability. New sentence is: Support for microprofile monitoring and configuration features (config, health, metrics, open-tracing).

Reply:
Thanks. If we ever add a layer that's meant to "provide _all_ microprofile API we support" then it can depend on observability. So doing it this way preserves flexibility.